bouncer/templates/base.html.jinja2

```diff
@@ -2,6 +2,19 @@
   <html lang=en>
   <head>
     <meta charset=utf-8>
+
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@hypothes_is" />
+    <meta name="twitter:title" content="{{ meta_title|safe }}" />
+    <meta name="twitter:description" content="Follow the link to see this annotation on the original page.  Use hypothes.is to annotate the Web and PDFs." />
```

Comment:
When the organization's name is written in text, as opposed to a logo, I believe it should be written as "Hypothesis".